sql/src/test/java/io/crate/integrationtests/disruption/discovery/MasterDisruptionIT.java

```diff
@@ -236,4 +244,51 @@ public void testVerifyApiBlocksDuringPartition() throws Exception {
         // the test to fail due to unfreed resources
         ensureStableCluster(2, nonIsolatedNode);
     }
+
+    @Test
+    @Repeat(iterations = 30)
```

Comment:
Do we really want to keep the repeat?

Reply:
No, it's just so we repeat the tests in the CI before we merge. Sorry should've mentioned the 2nd commit is a temporary one that will be dropped before merge.